networkx/algorithms/centrality/eigenvector.py

```diff
@@ -12,42 +12,64 @@
 def eigenvector_centrality(G, max_iter=100, tol=1.0e-6, nstart=None, weight=None):
     r"""Compute the eigenvector centrality for the graph `G`.
 
-    Eigenvector centrality computes the centrality for a node based on the
-    centrality of its neighbors. The eigenvector centrality for node $i$ is
-    the $i$-th element of the vector $x$ defined by the equation
+    Eigenvector centrality computes the centrality for a node by adding
+    the centrality of its predecessors. The centrality for node $i$ is the
+    $i$-th element of a left eigenvector associated with the positive
+    eigenvalue $\lambda$ of maximum modulus. Such an eigenvector $x$ is
```

Comment:
No, the Perron-Frobenius theorem cannot guarantee that there are no negative eigenvalues of maximum modulus, even in the strongly connected case. A graph with two nodes and two (non-loop) arcs is an example—it's strongly connected, and its eigenvalues are -1 and 1. You can only guarantee that there will be a positive eigenvalue of maximum modulus.

The phrase is not ambiguous because maximizing the modulus over all eigenvalues or all positive eigenvalues gives the same value—the spectral radius. But I'll change it to your suggestion.